src/components/templates/Playa/AvatarLayer.tsx

```diff
@@ -123,16 +123,28 @@ const AvatarLayer: React.FunctionComponent<PropsType> = ({
     };
   }, [user]);
 
+  const selfUserProfile = user?.uid
+    ? partygoers.find((pg) => pg.id === user.uid)
+    : undefined;
+
   const myAvatar = useMemo(
-    () => (
-      <MyAvatar
-        serverSentState={myServerSentState}
-        walkMode={walkMode}
-        sendUpdatedState={sendUpdatedState}
-        setMyLocation={setMyLocation}
-      />
-    ),
-    [myServerSentState, walkMode, sendUpdatedState, setMyLocation]
+    () =>
+      selfUserProfile ? (
+        <MyAvatar
+          serverSentState={myServerSentState}
+          walkMode={walkMode}
+          sendUpdatedState={sendUpdatedState}
+          setMyLocation={setMyLocation}
+          onClick={() => setSelectedUserProfile(selfUserProfile)}
```

Comment:
This will get swapped out soon for a menu allowing you to become available for chat, we can put the user profile there, would you be ok to not include this?